MIGRATION.md

```diff
@@ -99,9 +99,9 @@ func Get() (*Config, error) {
  }
 
  if cfg.Debug {
-  cfg.PatternLibraryAssetsPath = "http://localhost:9000/dist"
+  cfg.PatternLibraryAssetsPath = "http://localhost:9001/dist/assets"
  } else {
-  cfg.PatternLibraryAssetsPath = "//cdn.ons.gov.uk/sixteens/f80be2c"
+  cfg.PatternLibraryAssetsPath = "//cdn.ons.gov.uk/dp-design-system/{commit_id}"
```

Comment:
I prefer to have a working example, saves the developer having to infer what the `{commit_id}` is